MovieSharing/VideoModel.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  Video.swift
+//  MovieSharing
+//
+//  Created by Mariam Korashi on 28.09.21.
+//
+
+import Foundation
+
+struct VideosResponse:Decodable {
+    
+    var items: [Video]
+    
```

Comment:
remove empty lines at the end of structs/classes before the closing curly bracket